TrackletAlgorithm/TrackletProcessor.h

```diff
@@ -617,7 +553,10 @@ TC::processStubPair(
     success = TC::diskSeeding<Seed, InnerRegion, OuterRegion>(negDisk, innerStub, outerStub, &rinv, &phi0, &z0, &t, phiL, zL, &der_phiL, &der_zL, valid_proj, phiD, rD, &der_phiD, &der_rD, valid_proj_disk);
   // Write the tracklet parameters and projections to the output memories.
   const TrackletParameters tpar(phiRegion, stubIndex1, stubIndex2, rinv, phi0, z0, t);
-  if (success) trackletParameters->write_mem(bx, tpar, npar++);
+  if (success) {
+    npar++;
```

Comment:
@aryd It seems like `npar` is completely unused at this point. Can it be removed throughout?

Reply:
Yes - done.